src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -699,7 +699,7 @@ def create_snapshot(cmd, resource_group_name, snapshot_name, location=None, size
 
     location = location or _get_resource_group_location(cmd.cli_ctx, resource_group_name)
     if source_blob_uri:
-        option = DiskCreateOption.import_enum
+        option = DiskCreateOption.IMPORT
```

Comment:
Could we use lowercase to keep consistent with other case styles?
```suggestion
        option = DiskCreateOption.import
```